runtime/vm/LayoutFFITypeTable.cpp

```diff
@@ -0,0 +1,140 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2024
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+#include "j9.h"
+#include "j9protos.h"
+
+extern "C" {
+
+#if JAVA_SPEC_VERSION >= 16
+
+/**
+ * @brief Return the hash value for an entry in the layout string table.
+ *
+ * @param entry the layout string entry
+ * @param userData the data specified as userData when the hashtable was created,
+ *                 in our case pointer to the port library
+ * @returns the hash value
+ */
+static UDATA
+layouStrFFITypeHashFn(void *entry, void *userData)
```

Comment:
Is there a reason you omitted `t` from these function names (e.g. `layouStrFFITypeHashFn` instead of `layoutStrFFITypeHashFn`)? If not, please spell `layout` fully.

Reply:
Fixed the typo everywhere.